test/models/test_omnivore.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import unittest
+
+import torch
+import torchmultimodal.models.omnivore as omnivore
+from torchmultimodal.utils.common import get_current_device
+
+
+class TestOmnivoreModel(unittest.TestCase):
+    def setUp(self):
+        torch.manual_seed(42)
+        self.device = get_current_device()
+
+    def test_omnivore_swin_t_forward(self):
+        model = omnivore.omnivore_swin_t().to(self.device)
+        self.assertTrue(isinstance(model, torch.nn.Module))
+
+        image = torch.randn(1, 3, 1, 112, 112)  # B C D H W
+        image_score = model(image, input_type="image")
+        self.assertEqual(image_score.size(), torch.Size((1, 1000)))
```

Comment:
please use our test utility to assert on values and shapes. (we need to assert values too)

Reply:
Thanks for the suggestion! Will do.

Reply:
Have you found this?

https://github.com/facebookresearch/multimodal/blob/e8575411d8fdc3f97c51d36789b2f6c530704a50/test/test_utils.py#L74

Reply:
@langong347 after I read the utility `assert_expected` it seems to compare two tensor with float type. In this case, I think `assertEqual` is better for comparing the size since it is integer?